src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Controllers/PublicationsController.cs

```diff
@@ -40,4 +40,23 @@ public PublicationsController(IPublicationService publicationService)
                 search: request.Search)
             .HandleFailuresOrOk();
     }
+
+    /// <summary>
+    /// Get a publication’s details"
+    /// </summary>
+    /// <remarks>
+    /// Gets a specific publication’s details.
```

Comment:
Perhaps we should improve on this with: `Get a specific publication's summary details.`

Reply:
done